modules/openideal_idea/src/Plugin/votingapi_widget/FiveStarWidget.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Drupal\openideal_idea\Plugin\votingapi_widget;
+
+use Drupal\votingapi_widgets\Plugin\votingapi_widget\FiveStarWidget as BaseFiveStarWidget;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Assigns ownership of a node to a user.
+ *
+ * Copy of FiveStartWidget but with changed permissions.
```

Comment:
I think it might be possible to swap a plugin class without providing a new one. Please take a look at plugin manager class.

Reply:
I know that. But specially created a new one to give the ability to users use default one, without this permission logic.